deploy/terraform/variables.tf

```diff
@@ -542,6 +559,141 @@ variable "vpc_cidr_block" {
   default     = "10.0.0.0/16"
 }
 
+variable "allow_all_cidr_block" {
+  description = "CIDR block to allow all IP addresses"
+  type = string
+  default = "0.0.0.0/0"
+}
+
+variable "allow_all_ipv6_cidr_block" {
+  description = "IPV6 CIDR block to allow all IP addresses"
+  type = string
+  default = "::/0"
+}
+
+variable "tcp_network_protocol" {
+  description = "tcp protocol for VPC"
+  type = string
+  default = "TCP"
+}
+
+variable "allow_all_vpc_protocol" {
+  description = "Allow all protocol for VPC"
+  type = string
+  default = "-1"
+}
+
+variable "allow_all_vpc_port" {
+  description = "Allow all port for VPC"
+  type = number
+  default = 0
+}
+
+variable "mq_engine_type" {
+  description = "Message queue engine type"
+  type = string
+  default = "ActiveMQ"
+}
+
+variable "mq_engine_version" {
+  description = "Message queue engine version"
+  type = string
+  default = "5.16.3"
+}
+
+variable "mq_host_type" {
+  description = "Message queue host instance type"
+  type = string
+  default = "mq.t2.micro"
+}
+
+variable "dynamodb_table_hash_key" {
+  description = "Attribute to use as the dynamodb hash key"
+  type = string
+  default = "key"
+}
+
+variable "dynamodb_attribute_type_string" {
+  description = "Attribute type of string for dynamodb table"
+  type = string
+  default = "S"
+}
+
+variable "dynamodb_table_billing_mode" {
+  description = "Billing mode for the dynamodb table"
+  type = string
+  default = "PAY_PER_REQUEST"
+}
+
+variable "lb_health_check_matcher_port_range" {
+  description = "Response codes to use when checking for a healthy responses from health check"
+  type = string
+  default = "200-499"
+}
+
+variable "lb_health_check_poll_interval" {
+  description = "Approximate amount of time, in seconds, between health checks"
+  type = number
+  default = 300
+}
+
+variable "iam_policy_version" {
+  description = "Version for IAM policies"
+  type = string
+  default = "2012-10-17"
+}
+
+variable "iam_statement_assume_role" {
+  description = "Policy statement with the defaults for assume role"
+  type = list(object({
+    Action = string
+    Effect = string
+    Sid = string
+    Principal = object({
+      Service = string
+    })
+  }))
+  default = [
+    {
+      Action = "sts:AssumeRole"
+      Effect = "Allow"
+      Sid    = ""
+      Principal = {
+        Service = "ecs-tasks.amazonaws.com"
+      }
+    }
+  ]
+}
```

Comment:
This might make more sense as an [iam_policy_document](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document).

Reply:
completed and migrated all policy documents from within iam.tf into data source file for consistency